### PR TITLE
Added back overflow-y scroll to allow users to scroll through chat messages.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixes an issue where a user could not scroll through messages in Connect.

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [x] no, they are coming in a subsequent PR

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Bobby Moynahan on SNL in character smashing a computer saying "Fixed it!"

### CODE REVIEW

File: app/javascript/chat/chat.jsx [hunk 1]
```diff
@@ -1478,6 +1478,7 @@ export default class Chat extends Component {
             onDrop={this.handleImageDrop}
           >
             <div
+              data-testid="messages"
```
Comment (nickytonline) [inline app/javascript/chat/chat.jsx:1481]: Used for tests that will be coming shortly.

**Review by Zhao-Andy — APPROVED** [review]

**Comment by Zhao-Andy** [comment]
Oops early approval hehe, re-requested myself.

**Review by nickytonline — COMMENTED** [review]
I will add the test in a subsequent PR as I can test this in jest since we do not generate our CSS for the tests. I will look into doing a test in Capybara.

**Review by fdoxyz — APPROVED** [review]
The fix works well on both Chrome for Mac & mobile (native + web) locally 🙌🏼

**Review by Zhao-Andy — APPROVED** [review]
LGTM!

**Pull request merged**